### PR TITLE
Adding PluginHelp.Config to verify-owners plugin

### PR DESCRIPTION
**Pull request by matthyx**
ref #10714

### CODE REVIEW

File: prow/plugins/verify-owners/verify-owners.go [hunk 1]
```diff
@@ -46,8 +47,17 @@ func init() {
 }
 
 func helpProvider(config *plugins.Configuration, enabledRepos []string) (*pluginhelp.PluginHelp, error) {
+	configString := `The verify-owners plugin does not have any blacklisted label to watch for.`
```
Comment (stevekuznetsov) [inline prow/plugins/verify-owners/verify-owners.go]: This might be confusing for someone. Can we pass no config if nothing is configured?

**Comment by matthyx** [comment]
@stevekuznetsov can you review again?

**Pull request merged**